Lib/Protocols/IdFTPServer.pas

```diff
@@ -739,6 +739,26 @@   EIdFTPCannotBeNegative = class(EIdFTPServerException);
 
   TIdFTPServer = class;
 
+  TIdFTPServerInfo = class(TPersistent)
+  protected
+    FAdditionalFacts : TStrings;
+    FServerName : String;
+    FServerVersion : String;
+    FServerVendor : String;
+    FPlatformDescription : String;
+    FPlatformVersion : String;
+    procedure SetAdditionalFacts(const AValue: TStrings);
+  public
+    constructor Create;
+    destructor Destroy; override;
+  published
+    property ServerName : String read FServerName write FServerName;
+    property ServerVersion : String read FServerVersion write FServerVersion;
+    property ServerVendor : String read FServerVendor write FServerVendor;
+    property PlatformDescription : String read FPlatformDescription write FPlatformDescription;
+    property PlatformVersion : String read FPlatformVersion write FPlatformVersion;
+    property AdditionalFacts : TStrings read FAdditionalFacts write SetAdditionalFacts;

```

Comment:
Should `CaseSensitive` and `DirSeparator` be included as properties here, to match `TIdFTPServerIdentifier` in `IdFTP.pas`?